### ISSUE TITLE
How can I load a model finetuned from current model?

### ISSUE THREAD

**Issue by JoyHuYY1412** [facebookresearch/maskrcnn-benchmark#444]
## ❓ Questions and Help
In your code, I think the line  `    _ = checkpointer.load(cfg.MODEL.WEIGHT)` works as trained model loader, am I right? 

So if I want to , for example, add a layer in your model, train the new model and save a checkpoint, how can I load the .pth file I get in future work? Because I usually use the code like `
state_dict = torch.load('./ckp/checkpoint_10800.pth')
model.load_state_dict(state_dict)
`
to load model, I am a little confused. 
Thank you for your help.

**Comment by fmassa** [comment]
If you add an extra layer in the model, the `checkpointer` will skip a layer if it is not present in the saved file.
So you could use it right away I believe?

**Comment by JoyHuYY1412** [comment]
> If you add an extra layer in the model, the `checkpointer` will skip a layer if it is not present in the saved file.
> So you could use it right away I believe?

thanks. but after I get a new checkpoint, how to load it? do I need to change the Model.weight parameter?

**Comment by fmassa** [comment]
Yes, just change the `MODEL.WEIGHT` field in the config to point to the modified checkpoint.
